emperor/support_files/js/color-view-controller.js

```diff
@@ -182,6 +186,16 @@ ColorViewController.getDiscreteColor = function(index, map){
   return ColorViewController._discreteColormaps[map][index];
 };
 
+/* Change the color for a set of plottables - group: list of plottables */
```

Comment:
:+1: Recommend using the javascript doc-strings we've used in view-controller.js